static/js/reducers/email.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {
+  START_EMAIL_EDIT,
+  UPDATE_EMAIL_EDIT,
+  CLEAR_EMAIL_EDIT,
+} from '../actions/email';
+import type { Action } from '../flow/reduxTypes';
+
+export const INITIAL_EMAIL_STATE = {};
+
+export const NEW_EMAIL_EDIT = {
+  subject:    null,
+  body:       null,
+  query:      null,
+};
+
+export type EMAIL_EDIT_STATE = {
```

Comment:
it seems that in other parts of our code, we name state _types_ with camel case (eg: `UIState`, `ProfileState`), and _examples_ of those states with upper-snake-case (eg: `INITIAL_UI_STATE`). shouldn't `EMAIL_EDIT_STATE` be named `EmailEditState`? also, going by the example in static/js/reducers/ui.js, it seems that `NEW_EMAIL_EDIT` and `INITIAL_EMAIL_STATE ` should be declared after `EmailEditState`, and should be defined like so:

```javascript
export type EmailEditState { ... }

export const INITIAL_EMAIL_STATE: EmailEditState = { }

export const NEW_EMAIL_EDIT: EmailEditState = { ... }
```

let me know if i'm missing something here